src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -129,20 +129,52 @@ class AudioPipelineOutput(BaseOutput):
 
 
 def is_safetensors_compatible(filenames, variant=None) -> bool:
-    pt_filenames = set(filename for filename in filenames if filename.endswith(".bin"))
-    is_safetensors_compatible = any(file.endswith(".safetensors") for file in filenames)
-
-    for pt_filename in pt_filenames:
-        _variant = f".{variant}" if (variant is not None and variant in pt_filename) else ""
-        prefix, raw = os.path.split(pt_filename)
-        if raw == f"pytorch_model{_variant}.bin":
-            # transformers specific
-            sf_filename = os.path.join(prefix, f"model{_variant}.safetensors")
+    """
+    Checking for safetensors compatibility:
+    - By default, all models are saved with the default pytorch serialization, so we use the list of default pytorch
+      files to know which safetensors files are needed.
+    - The model is safetensors compatible only if there is a matching safetensors file for at least one default pytorch
+      file.
```

Comment:
Where do we look for this mapping? If it cannot be explained in one line, no need to include it then.

Reply:
> at least one

Isn't that all default pytorch files ?